core/dev-tools/simpledev.js

```diff
@@ -0,0 +1,109 @@
+     //                        ////                   //
+     //                     ////////                  //
+    //                    ////////   ///               //
+    //                 ////////   ////////             //
+   //                ////////   ////////   //           //
+  //              ////////   ////////    ///////        ///
+  //            ////////   ////////   ////////           //
+ //          ////////   ////////    ////////              //
+ //            ////////   ////   ////////                 //
+ //              ////////      ////////                   //
+  ///               ////////   /////                    ///
+    ///               ////////                         ///
+      ///                //////                     ///
+        ///                //                     ///
+          ///                                   ///
+            ///                               ///
+              ///                           ///
+                /////////////////////////////
+
+
+
+
+        ///                    ///               ///
+        ///
+        ///        ///    ///  ///   ////////    ///
+        ///        ///    ///  ///   ///   ///   ///
+        ///        ///    ///  ///    /////      ///
+        /////////  ////// ///  ///   /////////   ///
+                                    ///     ///
+                                      //////
+
+`;
+console.log('\x1b[32m', logo_ascii,'\x1b[0m');
+
+const rootPath = './dev-tools/simple-app';
+const indexPath = rootPath + '/index.html';
+try {
+  if (fs.existsSync(indexPath)) {
+    console.log('\x1b[32mFound ' + indexPath,'\x1b[0m');
+  } else {
+    console.log('\x1b[33mCould not find ' + indexPath, '\nInitializing new minimalistic Luigi app from template...','\x1b[0m');
+    if (!fs.existsSync(rootPath)) {
+      fs.mkdirSync(rootPath);
+    }
+    fs.copy('./dev-tools/templates/simple', rootPath);
+    console.log('\x1b[32mNew Luigi app created under ' + rootPath,'\x1b[0m');
+  }
+} catch(err) {
+  console.error(err)
+}
+
+webpack_compiler.hooks.watchRun.tap('CLI output', () => {
+  console.log('\x1b[33mWebpack [' + new Date().toLocaleTimeString() +']: ', ' Rebuild in progress...','\x1b[0m');
+});
+
+const watching = webpack_compiler.watch({
+  aggregateTimeout: 300,
+  poll: undefined,
+  logLevel: 'verbose'
+}, (err, stats) => {
+  if (err) {
+    console.error(err.stack || err);
+    if (err.details) {
+      console.error(err.details);
+    }
+    return;
+  }
+
+  if(stats.hasErrors()) {
+    console.log('\x1b[33mWebpack [' + new Date().toLocaleTimeString() +']: ','\x1b[31m', "Rebuild of Luigi core failed! \n",
+      stats.toString({
+      all: false,
+      errors: true,
+      colors: true,
+
+    }));
+  } else {
+    console.log('\x1b[33mWebpack [' + new Date().toLocaleTimeString() +']: ','\x1b[32m', 'Luigi core rebuilt without errors.\n','\x1b[0m');
+  }
+});
+
+var params = {
+	port: 4000,

```

Comment:
Could we move it away from 4000, since this is used for docu already?
```suggestion
	port: 4100,
```